src/screens/sync/Devices.tsx

```diff
@@ -136,66 +141,81 @@ const deviceListStyles = StyleSheet.create({
   },
 });
 
-const DeviceList = () => {
+export const DeviceList = ({
+  setAndOpenModal,
+}: {
+  setAndOpenModal: (content: TitleAndDescription) => void;
+}) => {
   const { formatMessage: t } = useIntl();
   const [shouldStart, setShouldStart] = React.useState(false);
+
   return (
-    <View>
-      <View style={deviceListStyles.headerRowContainer}>
-        <View style={deviceListStyles.headerTitleContainer}>
-          <Text size="medium" bold>
-            {t(m.localDevices)}
-          </Text>
-          <Spacer direction="horizontal" size={spacing.medium} />
-          <TouchableOpacity style={deviceListStyles.infoButton}>
-            <MaterialIcon name="help" size={14} color={colors.DARK_GRAY} />
-          </TouchableOpacity>
-        </View>
-        <Pressable
-          onPress={() => setShouldStart(true)}
-          android_ripple={{ radius: 100 }}
-          style={deviceListStyles.syncButton}
-        >
-          <View style={deviceListStyles.syncButtonContentContainer}>
-            <MaterialIcon
-              name="lightning-bolt"
-              size={20}
-              color={colors.WHITE}
-            />
-            <Spacer direction="horizontal" size={spacing.medium} />
-            <Text size="medium" color={colors.WHITE} bold>
-              {t(m.sync)}
+    <React.Fragment>
```

Comment:
nit: fragment not needed here

Reply:
https://github.com/digidem/mapeo-mobile-shell/pull/28/commits/f63808067cd2927413d7e868415aa0a2fcafec8c